File: factorial.c
#include <stdio.h>
#include <time.h>
long double term (int n)
{long double fact=1;
if(n==1)
	{
	return 1;}
fact=n*term(n-1);
return fact;
}
int main() {
  float start=clock();
printf("%Lf",term(20));
 float end=clock();
 printf("time for C's Factorial program %f /n",(end-start)/CLOCKS_PER_SEC);
return 0;
}

File: factorial.py
from scipy.special import factorial
from time import time
def term(n):
    if (n==1):
        return 1
    fact=1
    fact=n*term(n-1)
    return fact
start=time()
print (term(20))
end=time()
print ("execu.time for python's user factorial\n",end-start)
start=time()
print (factorial(20))
end=time()
print ("execu.time for python's scipy factorial\n",end-start)
